### PR TITLE
Layout_Product-cards

### PR DESCRIPTION
**Pull request by stunnerboyone**
- [DEMO LINK](https://stunnerboyone.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://stunnerboyone.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,56 @@
...
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-amount">$2,199</div>
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover">
```
Comment (denys-danyliuk) [inline src/index.html:54]: 14'th point in checklist

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,56 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review-text">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-amount">$2,199</div>
```
Comment (denys-danyliuk) [inline src/index.html:48]: use p, 12'th point in checklist

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,56 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <!-- <div class="card__image"></div> -->
+      <img
+        src="/images/imac.jpeg"
+        alt="imac"
```
Comment (denys-danyliuk) [inline src/index.html:22]: 13'th point in checklist

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,56 @@
...
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <!-- <div class="card__image"></div> -->
```
Comment (denys-danyliuk) [inline src/index.html:19]: delete comments

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by mykhalenych — APPROVED** [review]
Screenshot 2023-08-07 at 17 30 08

don't push files and folders than aren't related to review

**Pull request closed without merging**